www/includes/easyparliament/templates/html/section/_section_content.php

```diff
@@ -51,7 +51,7 @@
         $body = preg_replace('#<p>\s*</p>#', '', $body);
 
         # Assume a paragraph starting with a lowercase character should be run on
-        $body = preg_replace('#(?<!:)</p>\s*<p[^>]*>(?=[a-z])(?![ivx]+\.)#', ' ', $body);
+        $body = preg_replace('#(?<!:|\])</p>\s*<p[^>]*>(?=[a-z])(?![ivx]+\.)#', ' ', $body);
```

Comment:
*looks up negative lookbehind docs*

Might it be easier to read with a character class `[:\]]` rather than alternation `:|\]`? (I think that's equivalent in this case?)